build.gradle

```diff
@@ -6,7 +6,7 @@ def ext = rootProject.ext
 
 buildscript {
     ext {
-        sdkVersion = "8.0.0"
+        sdkVersion = "8.0.2"
```

Comment:
Is suppose this relates to your local setup but next version should probably be 8.1.0 ? Not sure how we should handle this, maybe set the probable next version in the SDK repo, so that it creates an appropriate snapshot when merged, and use that snapshot here? @axuan25

Reply:
Totally agree and didn't really tilt, thanks @nbrr. In semver, new functionality with no breaking change = minor version 🙇

Reply:
I am not sure the changes brought by the ticket 1781 should be included in a new major version because it doesn't introduce any breaking change.